code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -77,5 +77,33 @@
 		return TRUE
 	return ..()
 
+/datum/species/pod/create_pref_unique_perks()
+	var/list/to_add = list()
+
+	to_add += list(list(
+		SPECIES_PERK_TYPE = SPECIES_NEUTRAL_PERK,
+		SPECIES_PERK_ICON = "lightbulb",
+		SPECIES_PERK_NAME = "Photosynthetic",
+		SPECIES_PERK_DESC = "As long as you are concious, and within a well-lit area, you will slowly heal brute, burn, toxin and oxygen damage and gain nutrition - and never get fat! \
+		However, if you are LOW on nutrition, you will progressively take brute damage until you die or enter the light once more."
```

Comment:
should mention that being in darkness will drain your nutrition (unless I'm misreading the code here)

also, going into the light won't IMMEDIATELY make you stop taking damage, it'll just start giving you nutrition, which will help bring you above the "you take damage below this point" threshold

Reply:
Passive nutrition loss is expected for all species so doesnt need to be stated

Also true, but Im nto sure thats important enough to be stated

If I get the energy I'll include it tho